### ISSUE TITLE
Maps is unable to scan picture location when encryption is enabled.

### ISSUE THREAD

**Issue by Bigdrivernet** [nextcloud/maps#549]
In my Nextcloud Instance the Maps app does not display my Images. When I try to rescan Images via occ following occurs:

root@h2917249:~# nextcloud.occ maps:scan-photos
Encryption is enabled. Aborted.

**Comment by tacruc** [comment]
Wouldn't it be scary, if our app could extract coordinates out of encrypted pictures?

**Comment by Bigdrivernet** [comment]
Yes but I would expect it to work for the user that's logged in. And that's not working.

**Comment by simbelmas** [comment]
The "encryption" is not precise nextcloud context.
Two encryption types can be used:

### End to end encryption
When using end to end encryption, server cannot decrypt file, which is wanted.
There is no risk about trying to index them as the only thing the app will find is a file with unknown content.

### Server side encryption
As the app is part of nextcloud and the server can decrypt all users files to display them like in files app, i think this is not as scary as it seems. 
In addition, app only shows connected user data.

Is it possible to let the app index files readable on server side and skip end-to-end encrypted ones?

**Comment by marekful** [comment]
Can we have an update on this? Maybe it's fixed in v23 and Maps can read geotags of photos when when server-side encryption is used?

**Comment by softcoder** [comment]
This cant be the only plugin, i mean gpxmod reads track files on encrypted servers, can we get this fixed?

**Comment by softcoder** [comment]
FYI I played with the code and disabling the encryption check makes everything work fine, I am able to rescan both photos and tracks!

**Comment by gwenl** [comment]
It seems photo scanning is disabled altogether when server encryption is enabled, even when there are photos on an unencrypted read-only external storage. IMO the app should at least try scanning those.

**Comment by nils-one** [comment]
> FYI I played with the code and disabling the encryption check makes everything work fine, I am able to rescan both photos and tracks!

Can you give a hint where this check takes places? Just to save some time for someone who also wants to play with this.

**Comment by peterablehmann** [comment]
Personally, this issue doesn't apply to me anymore. My instance just scans as supposed with server side encryption enabled.

**Comment by jkhsjdhjs** [comment]
The issue still applies to me with nextcloud 26.0 and server-side encryption enabled.
